.travis.yml

```diff
@@ -15,7 +15,17 @@ matrix:
     - os: linux
       rust: nightly
     - os: osx
-      osx_image: xcode11
+      osx_image: xcode11.2
+      addons:
+        chrome: stable
+        firefox: latest
+      install:
+        - curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | sh
+        - source ~/.nvm/nvm.sh
+        - nvm install v12.13.1
```

Comment:
If you don't care about specific version and you want to use latest LTS, you can replace this line with ...

`nvm install --lts`

... and it will install latest LTS available.